accepted/000070-cluster_awareness.md

```diff
@@ -0,0 +1,272 @@
+- Feature Name: MVP Cluster awareness for Uyuni/SUSE Manager
+- Start Date: 2020-02-20
+- RFC PR: https://github.com/uyuni-project/uyuni-rfc/pull/36
+
+# Summary
+[summary]: #summary
+
+Make Uyuni/SUSE Manager aware and capable of deploying and managing clusters.
+
+# Motivation
+[motivation]: #motivation
+
+<!-- - Why are we doing this?
+- What use cases does it support?
+- What is the expected outcome?
+
+Describe the problem you are trying to solve, and its constraints, without coupling them too closely to the solution you have in mind. If this RFC is not accepted, the motivation can be used to develop alternative solutions. -->
+
+Uyuni/SUSE Manager should be able to interact with other SUSE products:
+- SUSE Container as a Service Platform (CaaSP) and Cloud Application Platform (CAP). From now on, we consider CAP as an equivalent of CaaSP for our purposes
+- SUSE Enterprise Storage (SES)
+- SUSE Linux Enterprise High Availability (SLE-HA) and SUSE Linux Enterprise HA for SAP Products (SHAP)
```

Comment:
From what I understand, we're focusing on the toolchain. So non-SUSE Ceph should be easy to support on this level. Kubernetes is managed through skuba for now (which technically isn't SUSE-specific, but we are the only users), but it would be possible to replace that integration with one that uses plain kubeadm or Cluster API once those are more mature.

Reply:
Other cluster "products" can be supported by following the principles in this RFC: provide a cluster manager and hook it up to the basics actions we outlined.